src/toolkit/utils/index.js

```diff
@@ -3,3 +3,16 @@ export { searchNote } from "./searchNote";
 export { getLabels } from "./getLabels";
 export { getLabelSelected, sort } from "./getFilters";
 export { usePageViewTracker } from "./usePageViewTracker";
+export { loginService, signupService } from "./authService";
+export {
+  getNoteService,
+  addNoteService,
+  updateNoteService,
+  deleteNoteService,
+  archiveNoteService,
+  unArchiveNoteService,
```

Comment:
suggestion: to avoid many imports and exports, you can export all the methods as one object and use that object and method name while invoking